src/components/AdminPortal/EventManagement/AgendaItemsDeleteModal/AgendaItemsDeleteModal.spec.tsx

```
@@ -0,0 +1,42 @@
+    const closeBtn = screen.getByTestId('deleteAgendaItemCloseBtn');
+    fireEvent.click(closeBtn);
+    expect(mockToggleDeleteModal).toHaveBeenCalled();
+  });
+
+  it('should call deleteAgendaItemHandler when confirm button is clicked', () => {
+    render(<AgendaItemsDeleteModal {...defaultProps} />);
+    const confirmBtn = screen.getByTestId('deleteAgendaItemBtn');
+    fireEvent.click(confirmBtn);
+    expect(mockDeleteAgendaItemHandler).toHaveBeenCalled();
+  });
+});
```

ESLint

[error] Test file uses mocks (vi.fn()) but is missing afterEach cleanup. Add afterEach(() => { vi.clearAllMocks(); }) to prevent mock leakage.

(vitest-isolation/require-aftereach-cleanup)